src/main/java/net/sf/jabref/logic/xmp/XMPSchemaBibtex.java

```diff
@@ -96,59 +98,36 @@ public void setPersonList(String field, String value) {
         }
     }
 
-    @Override
     public String getTextProperty(String field) {
-        return super.getTextProperty(makeProperty(field));
+        return super.getUnqualifiedTextProperty(makeProperty(field)).getStringValue();
```

Comment:
I would completely remove that code and all related  as a) it is inconsistent and uncessary b)  From the javadoc: getUnqualified...() does not need the namespace to acces the property. However, with calling MakeProperty, you are explicit adding a namespace. c) Callin these  methods on the class would  automatically call the super methods via polymorphism